src/gwt/src/org/rstudio/core/client/theme/res/ThemeResources.java

```diff
@@ -297,4 +297,10 @@
    
    @Source("browsePackage_2x.png")
    ImageResource browsePackage2x();
+
+   @Source("suspended.png")
+   ImageResource suspended();
+
+   @Source("SuspendBlocked.png")
+   ImageResource suspendBlocked();
```

Comment:
Check file name here. The added .png is lower case for me -- so this won't load anything.
```suggestion
   @Source("suspendBlocked.png")
   ImageResource suspendBlocked();
```